daemon-systemd.in

```diff
@@ -10,7 +10,7 @@ PIDFile=@lockfile@
 PermissionsStartOnly=true
 ExecStartPre=-/usr/bin/mkdir /var/run/naemon
 ExecStartPre=/usr/bin/chown -R naemon:naemon /var/run/naemon/
-ExecStartPre=@bindir@/naemon -vp @pkgconfdir@/naemon.cfg
+ExecStartPre=su - naemon -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
```

Comment:
Is this because you want to start naemon as the user naemo? If so, systemd will start the process as the user specified in `User=` which is naemon.